monai/transforms/utils.py

```diff
@@ -103,6 +106,10 @@
     "generate_spatial_bounding_box",
     "get_extreme_points",
     "get_largest_connected_component_mask",
+    "get_largest_connected_component_mask_point",
```

Comment:
discussed offline with @KumoLiu , this function may be combined with the above one. I will figure out in this PR

Reply:
will still keep both, this function has different usages